app/workflows/ihp_tasks_factory.rb

```
@@ -6,6 +6,8 @@ def initialize(parent)
   end
 
   def create_ihp_tasks!
+    return unless @parent
```

Now because of activated callbacks, lots of tests are calling this factory and a lot of tasks don't have parents in the tests. I think creating a bunch parents in order for this to work is an overkill.

---

It's not clear to me why callbacks are activated now and not before?

Could we change the factorybot classes in the tests to always assign a parent?

I worry that silently returning here in the code to reflect a pattern in the tests will bite us eventually.

---

To always a assign a parent causes other issues. For example, to always assign a parent to a hearing_task in the factory itself breaks a ton of tests because hearing_task has a root task as its parent. So if there is already a root task or a root task is created later, it throws an error.

---

I am going to merge this PR the way it is and I will think about this problem.